lib/elixir/src/elixir_try.erl

```diff
@@ -23,8 +23,8 @@ each_clause({'catch', Meta, Raw, Expr}, S) ->
 
   Condition = [{'{}', Meta, Final}],
   {TC, TS} = elixir_clauses:clause(Meta, fun elixir_translator:translate_args/2,
-                                   Condition, Expr, Guards, S),
-  {[TC], TS};
+                                   Condition, Expr, Guards, S#elixir_scope{extra=nil}),
+  {[TC], TS#elixir_scope{extra=S#elixir_scope.extra}};
```

Comment:
Do you know why we need to change it here if we already changed it here? https://github.com/elixir-lang/elixir/pull/4504/files#diff-dd6727eaeac58aaeae8719869f854c8bR105